ldms/src/sampler/rdc_sampler/rdcinfo.c

```diff
@@ -81,7 +82,39 @@ static rdc_field_t default_field_ids[] = {
 	RDC_EVNT_XGMI_1_THRPUT
 };
 
-static const uint32_t num_fields_default = sizeof(default_field_ids)/sizeof(default_field_ids[0]);
+static const uint32_t num_fields_default_20 = sizeof(default_field_ids_20)/sizeof(default_field_ids_20[0]);
+
+/* expansion for metrics=v10 option */
+const char *default_metrics_10 = 
+	"RDC_FI_GPU_CLOCK,"
+	"RDC_FI_MEM_CLOCK,"
+	"RDC_FI_MEMORY_TEMP,"
+	"RDC_FI_GPU_TEMP,"
+	"RDC_FI_POWER_USAGE,"
+	"RDC_FI_PCIE_TX,"
+	"RDC_FI_PCIE_RX,"
+	"RDC_FI_GPU_UTIL,"
+	"RDC_FI_GPU_MEMORY_USAGE,"
+	"RDC_FI_GPU_MEMORY_TOTAL,"
+	"RDC_FI_ECC_CORRECT_TOTAL,"
+	"RDC_FI_ECC_UNCORRECT_TOTAL";
+
+/* expansion for metrics=v20 option */
+const char *default_metrics_20 = 
```

Comment:
XGMI is a fast bus between GPUs. There is a physic bus related to this and user can choose to use it or not even for Vega20+ cards. I think to name this as default_metrics-xgmi is better.

Reply:
@miketxli  can you suggest appropriate alternatives to both the abbreviations accepted by the metrics= argument? currently it's v10, v20. Sounds to me like maybe it should be (something_else_for_v10),xgmi instead. I can then make the c code match appropriately.

Reply:
I would say with_xgmi and without_xgmi, or xgmi and non_xgmi something like that. XGMI is introduced since Vega20. But not all new cards do support XGMI. Even for Vega20, there is an optional piece of HW to interconnect GPUs. If user choose to not connect. There will still be errors with XGMI.